storage/rocksdb/ha_rocksdb.cc

```diff
@@ -459,6 +459,7 @@ static uint32_t rocksdb_table_stats_sampling_pct;
 static my_bool rocksdb_enable_bulk_load_api = 1;
 static my_bool rocksdb_print_snapshot_conflict_queries = 0;
 static my_bool rocksdb_large_prefix = 0;
+static my_bool rocksdb_persist_auto_increment = 0;
```

Comment:
hmm, I even wonder if we should really have this sysvar since this diff is almost a bug fix rather than a feature. if this diff is tested properly, we might not want to pollute the code with extra knobs.